v2_papers/2010/lim_internet.json

```diff
@@ -0,0 +1,285 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Internet Traffic Classification Demystified: On the Sources of the Discriminative Power",
+    "authors": [
+      "Lim, Yeon-sup",
+      "Kim, Hyun-chul",
+      "Jeong, Jiwoong",
+      "Kim, Chong-kwon",
+      "Kwon, Ted “Taekyoung\"",
+      "Choi, Yanghee"
+    ],
+    "publication_name": "Co-NEXT '10 Proceedings of the 6th International COnference",
+    "publication_type": "peer_reviewed_conference",
+    "year": 2010,
+    "organization_publishers": [
+      "acm"
+    ],
+    "pages_number_of": 12,
+    "bibtex": {
+      "type": "inproceedings",
+    },
```

Comment:
The problem with the verifier is here. You can't have a comma after the last item in an object (meaning, this isn't valid JSON).
Even then, I'd add these fields:
"booktitle": "Proceedings of the 6th International COnference",
"organization": "ACM"